### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by AlexZhang21**
Code Submission on Practical 5 
Please review: @Liu1xuan, @Byeclouds, @Rinnki97

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        emails_to_names[email] = name
+
+        email = input("Email: ")
+
+    # Loop
+    for email, name in emails_to_names.items():
+        print(f"{name} ({email})")
+
+
+email_check()
```
Comment (Rinnki97) [inline prac_05/emails.py:42]: missing main function

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Estimate: 5 minutes
```
Comment (Rinnki97) [inline prac_05/hex_colours.py:1]: missing docstring for estimation!! which is going to make the code break down!

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+"""
+CP1404/CP5632 Practical
+Wimbledon
+Estimate: 30 minutes
+Actual:   40 minutes
+"""
+
+import csv
```
Comment (Rinnki97) [inline prac_05/wimbledon.py:8]: module that we didn't learn

File: prac_05/wimbledon.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+from collections import Counter
```
Comment (Rinnki97) [inline prac_05/wimbledon.py:9]: same

File: prac_05/word_occurrences.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        else:
+            word_counts[word] = 1
+
+    #get the length of longest word
+    longest_word_length = max(len(word) for word in word_counts.keys())
+
+    for word, count in sorted(word_counts.items()):
+        print(f"{word:{longest_word_length}} : {count}")
+
+word_count()
```
Comment (Rinnki97) [inline prac_05/word_occurrences.py:29]: missing main function

File: prac_05/emails.py (hunk 1 above)
Comment (Byeclouds) [inline prac_05/emails.py:42]: it should have main function

File: prac_05/state_names.py [hunk 6]
```diff
@@ -2,17 +2,28 @@
...
 File needs reformatting
+Estimate: 5 minutes
+Actual:   5 minutes
 """
 
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
-            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-print(CODE_TO_NAME)
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
```
Comment (Byeclouds) [inline prac_05/state_names.py:10]: repeat

File: prac_05/wimbledon.py (hunk 3 above)
Comment (Byeclouds) [inline prac_05/wimbledon.py:8]: we don't use the moudle we didn't learn

**Pull request merged**